src/porepy/models/poromechanics.py

```diff
@@ -261,6 +267,18 @@ def _is_nonlinear_problem(self) -> bool:
         """The coupled problem is nonlinear."""
         return True
 
+    def set_nonlinear_discretizations(self) -> None:
+        """Collect discretizations for nonlinear terms."""
+        # Nonlinear discretizations for the fluid mass balance subproblem. The momentum
+        # balance does not have any.
+        super().set_nonlinear_discretizations()
+        # Aperture changes render permeability variable. This requires a re-discretization
+        # of the diffusive flux in subdomains where the aperture changes.
```

Comment:
```suggestion
        # Aperture changes render permeability variable. This requires a 
        # re-discretization # of the diffusive flux in subdomains where the aperture
        # changes.
```

Reply:
What's going on here?

Reply:
I think you were above the 88 lines. If not, disregard.